src/System.Drawing.Common/src/System/Drawing/Printing/InvalidPrinterException.cs

```diff
@@ -22,6 +23,11 @@ public InvalidPrinterException(PrinterSettings settings)
             _settings = settings;
         }
 
+        protected InvalidPrinterException(SerializationInfo info, StreamingContext context) : base(info, context)
```

Comment:
If Mono uses this file InvalidPrinterException.Serializable.cs together they will have a duplicate method signature and the compiler will emit an error. I would put that method into a NonSerializable file like you did for the others.